deploy/controller/operator.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: multicluster-integrations
+  namespace: open-cluster-management
+  labels:
+    name: multicluster-integrations
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: multicluster-integrations
+  template:
+    metadata:
+      labels:
+        name: multicluster-integrations
+    spec:
+      serviceAccountName: multicluster-operators
+      volumes:
+      - name: multicluster-integrations-syncresource 
+        emptyDir: {}
+      containers:
+      - name: multicluster-integrations-syncresource
+        image: quay.io/open-cluster-management/multicloud-integrations:latest
+        imagePullPolicy: Always
+        command:
+        - /usr/local/bin/gitopssyncresc
+        - --appset-resource-dir=/etc/gitops-resources
+        env:
+        - name: WATCH_NAMESPACE
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: DEPLOYMENT_LABEL
+          value: multicluster-integrations-syncresource
+        - name: OPERATOR_NAME
+          value: multicluster-integrations
+        livenessProbe:
+          exec:
+            command:
+            - ls
+          initialDelaySeconds: 15
+          periodSeconds: 15
+        readinessProbe:
+          exec:
+            command:
+            - ls
+          initialDelaySeconds: 15
+          periodSeconds: 15
+        resources:
+          limits:
+            cpu: 100m
+            memory: 512Mi
+          requests:
+            cpu: 25m
+            memory: 64Mi
+        volumeMounts:
+        - mountPath: /etc/gitops-resources
+          name: multicluster-integrations-syncresource
+          readOnly: false
+      - name: multicluster-integrations-aggregation
+        image: quay.io/open-cluster-management/multicloud-integrations:latest
+        imagePullPolicy: Always
+        command:
+        - /usr/local/bin/pullmodel-aggregation
```

Comment:
I missed this part when changing the controller to multiclusterstatusaggregation. This should be `- /usr/local/bin/multiclusterstatusaggregation`